### PR TITLE
Add Milanote and WEBTOON

### PR DESCRIPTION
**Pull request by ClaraCF**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -12027,6 +12038,17 @@
...
 
+    {
+        "name": "WEBTOON",
+        "url": "https://www.webtoons.com/en/account/delete",
+        "difficulty": "easy",
+        "notes": "Check the \"I understand and want to delete my account\" option and click on \"Delete my account\". Note that any comments you wrote will not be deleted automatically so you will need to go <a href=\"https://www.webtoons.com/en/mycomment\">here</a> and delete them yourself.",
+        "domains": [
+            "www.webtoons.com"
+        ]
+
```
Comment (tupaschoal) [inline _data/sites.json:12049]:
```suggestion
```

File: _data/sites.json [hunk 2]
```diff
@@ -12636,3 +12658,4 @@
         ]
     }
 ]
+
```
Comment (tupaschoal) [inline _data/sites.json:12661]:
```suggestion
```

**Review by tupaschoal — COMMENTED** [review]
Hello, and thanks for submitting a PR to improve our website!

There's one problem on the sorting. According to our CI:

> Keys must be in alphanumeric order. webtoon needs to come after webex - free account

There are a couple line breaks that can be removed too. Can you please tend to those issues?

**Comment by ClaraCF** [comment]
Sorry, my bad. I think I fixed it now.

**Review by tupaschoal — COMMENTED** [review]
`WEBTOON` still needs to come after `WebTickets`, after that I think it will be fine

**Comment by ClaraCF** [comment]
Sorry, I misread.

**Review by tupaschoal — APPROVED** [review]
Don't worry, it's all fine now :)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**